paddle/fluid/framework/new_executor/interpretercore.cc

```diff
@@ -837,6 +877,19 @@ void InterpreterCore::ExecuteInstructionList(
 
   for (size_t i = 0; i < dependecy_count_.size(); ++i) {
     if (dependecy_count_[i] == 0) {
+      // NOTE(zhiqiu): hot fix for jit input var
+      platform::DeviceContextPool& pool =
+          platform::DeviceContextPool::Instance();
+      auto* default_dev_ctx = pool.Get(place_);
```

Comment:
Better to get the default_dev_ctx inside the following `if` statement, since we only need to obtain it in a few scenarios.